operator/cmd/mesh/manifest-versions.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 const (
-	versionsMapURL = "https://raw.githubusercontent.com/istio/operator/master/data/versions.yaml"
+	versionsMapURL = "https://raw.githubusercontent.com/istio/istio/master/operator/data/versions.yaml"
```

Comment:
Woah.... we are pulling this from github??? this is not a good idea

Reply:
I agree, what's your suggestion? 

make it as a built-in VFS file?

Reply:
I don't think anything should be in VFS presonally, but while we have everything else in VFS then this should be as well?